Exiled.CustomItems/API/Features/CustomWeapon.cs

```diff
@@ -60,6 +60,11 @@ public override ItemType Type
         /// </summary>
         public virtual byte ClipSize { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value whether or not to allow friendly fire with this weapon on FF-enabled servers.
+        /// </summary>
+        public virtual bool AllowFF { get; set; }
```

Comment:
This should be `AllowFriendlyFire` or just `FriendlyFire` as currently it will serialize in config as `allow_f_f:` which is ugly.